### PR TITLE
Update client to v0.2.0

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: src/components/Proposal/Create/SchemeForms/CreateSchemeRegistrarProposal.tsx [hunk 1]
```diff
@@ -308,7 +308,7 @@ class CreateSchemeRegistrarProposal extends React.Component<IProps, IState> {
                       >
                         <option value="">Select a scheme...</option>
                         {schemes.map((scheme, _i) => {
-                          return <option key={`edit_scheme_${scheme.address}`} value={scheme.address}>{schemeNameAndAddress(scheme.address)}</option>;
+                          return <option key={`edit_scheme_${scheme.staticState.address}`} value={scheme.staticState.address}>{schemeNameAndAddress(scheme.staticState.address)}</option>;
```
Comment (tibetsprague) [inline src/components/Proposal/Create/SchemeForms/CreateSchemeRegistrarProposal.tsx]: tell me more about staticState

File: src/components/Proposal/ProposalHistoryRow.tsx [hunk 2]
```diff
@@ -15,16 +13,18 @@ import { proposalFailed, proposalPassed } from "reducers/arcReducer";
 import { closingTime } from "reducers/arcReducer";
 import { IProfileState } from "reducers/profilesReducer";
 import { combineLatest, of } from "rxjs";
+// import { concatMap } from "rxjs/operators";
```
Comment (tibetsprague) [inline src/components/Proposal/ProposalHistoryRow.tsx]: remove

File: src/components/Proposal/ProposalHistoryRow.tsx [hunk 3]
```diff
@@ -207,27 +207,38 @@ const ConnectedProposalHistoryRow = connect<IStateProps, IDispatchProps, IContai
 
 export default (props: { proposal: Proposal; daoState: IDAOState; currentAccountAddress: Address}) => {
   const proposal = props.proposal;
-  const observable = combineLatest(
-    proposal.state(),
-    props.currentAccountAddress ? proposal.stakes({ where: { staker: props.currentAccountAddress}}) : of([]),
-    props.currentAccountAddress ? proposal.votes({ where: { voter: props.currentAccountAddress }}) : of([])
-  );
+  let observable;
+  if (props.currentAccountAddress) {
```
Comment (tibetsprague) [inline src/components/Proposal/ProposalHistoryRow.tsx]: this should be if !props.currentAccountAddress

File: src/components/Dao/DaoSchemesPage.tsx [hunk 4]
```diff
@@ -81,16 +80,22 @@ export default (props: {} & RouteComponentProps<any>) => {
 
   const observable = combineLatest(
     arc.dao(daoAvatarAddress).state(), // DAO state
-    arc.dao(daoAvatarAddress).schemes()
+    // eslint-disable-next-line
```
[the remark this answers is not in the document]
Comment (jellegerbrandy) [reply src/components/Dao/DaoSchemesPage.tsx]: it the rule that we need to use camelcase, it complains about the `not_in`. That is probably something we want to enforce for our own code, but not when we have to call outside functions, so afaic disabling it is ok here.

File: src/components/Proposal/ProposalHistoryRow.tsx (hunk 3 above)
Comment (jellegerbrandy) [reply src/components/Proposal/ProposalHistoryRow.tsx]: oops

File: src/components/Proposal/Create/SchemeForms/CreateSchemeRegistrarProposal.tsx (hunk 1 above)
Comment (jellegerbrandy) [reply src/components/Proposal/Create/SchemeForms/CreateSchemeRegistrarProposal.tsx]: I will :)

File: src/components/Dao/DaoSchemesPage.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (dkent600) [reply src/components/Dao/DaoSchemesPage.tsx]: but one should not be disabling eslint altogether, one should disable *just* the rule

**Pull request merged**